### PR TITLE
add tutorial for Helix cloud support

### LINKED ISSUE #1145: Add tutorial for cloud support
Add tutorial on how to use participant auto registration in cloud environment.

### PR DESCRIPTION
**Pull request by zhangmeng916**
### Issues

- [X] My PR addresses the following Helix issues and references them in the PR description:

Fixes #1145 

### Description

- [X] Here are some details about my PR, including screenshots of any UI changes:

This PR adds tutorial on how to use Helix cloud support. Specifically, it guides user through the steps to realized participant auto registration to a cluster.

### Tests

- [X] The following is the result of the "mvn test" command on the appropriate module:

mvn site:site
mvn site:stage
passed
Generated webpage is checked.

### Commits

- [X] My commits all reference appropriate Apache Helix GitHub issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Code Quality

- [X] My diff has been formatted using helix-style.xml 
(helix-style-intellij.xml if IntelliJ IDE is used)

### CODE REVIEW

**Comment by pkuwm** [comment]
@zhangmeng916 Good doc! I would suggest using google docs or office to do spell check if not yet. It may help polish the doc.

**Comment by zhangmeng916** [comment]
This PR is read to be merged. Approved by @pkuwm 
Final commit message:
Add tutorial for how to use cloud support in Helix for participant auto registration.

**Comment by kaisun2000** [comment]
A question, for .md, what command do you use to validate the format of it? "man"

**Comment by zhangmeng916** [comment]
> A question, for .md, what command do you use to validate the format of it? "man"

@kaisun2000 you can mvn site:site, then it'll give you the link to see the webpage generated by your md file.

File: website/1.0.1/src/site/markdown/tutorial_cloud_support.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+```
+$ curl -X PUT -H "Content-Type: application/json" http://localhost:1234/admin/v2/clusters/myCluster?addCloudConfig=true -d '
+{
+    "simpleFields" : 
+    {
+        "CLOUD_ENABLED" : "true",
+        "CLOUD_PROVIDER": "AWS",
+        "CLOUD_ID" : "12345"
+        "CLOUD_INFO_SOURCE": {"http://169.254.169.254/"}
+        "CLOUD_INFO_PROCESSOR_NAME": "AWSCloudInformationProcesser"
```
Comment (kaisun2000) [inline website/1.0.1/src/site/markdown/tutorial_cloud_support.md:86]: what is CLOUND_INFO_PROCESSOR_NAME used for in Helix?

Comment (zhangmeng916) [reply website/1.0.1/src/site/markdown/tutorial_cloud_support.md:86]: @kaisun2000 Helix implemented processor for Azure now, and the name is `AzureCloudInstanceInformationProcessor`

**Pull request merged**